cli/src/command/account.rs

```diff
@@ -937,9 +937,11 @@ async fn print_address(account: &Account, address: &Bip44Address) -> Result<(),
     let mut output_ids: &[OutputId] = &[];
     let mut amount = 0;
     let mut native_tokens = NativeTokensBuilder::new();
-    let mut nfts = Vec::new();
     let mut accounts = Vec::new();
     let mut foundries = Vec::new();
+    let mut nfts = Vec::new();
+    let mut delegations = Vec::new();
+    let mut anchors = Vec::new();
```

Comment:
Do we even want to sync/display them if we don't support to consume them?

Reply:
I think so, yeah